pkg/config/config.go

```diff
@@ -1101,12 +1101,6 @@ func Reload() (*Config, error) {
 }
 
 func (c *Config) ActiveDestination() (uri, identity string, machine bool, err error) {
-	if uri, found := os.LookupEnv("CONTAINER_HOST"); found {
-		if v, found := os.LookupEnv("CONTAINER_SSHKEY"); found {
-			identity = v
-		}
-		return uri, identity, false, nil
-	}
 	connEnv := os.Getenv("CONTAINER_CONNECTION")
```

Comment:
Should we also move the CONTAINER_CONNECTION env var to not have precedence over the `--connection` flag?

Reply:
Yes

Reply:
Very good catch!

We should also add tests.  Look at the following example for instance: https://github.com/containers/common/blob/main/pkg/config/config_local_test.go#L512-L522

Once a config (that we can add to the git tree) is loaded, we can call `ActiveDestination` and test which value is being returned with and without the two CONTAINER_ variables being set in the environment.

That will make sure that the API behaves as expected and that we won't silently regress on it in the future.